frontend/test/metabase-lib/lib/queries/StructuredQuery-clean.unit.spec.js

```diff
@@ -145,22 +140,6 @@ describe("StructuredQuery", () => {
         const q = ordersTable.query().nest();
         expect(q.clean().query()).toEqual({ "source-table": ORDERS_ID });
       });
-
-      it("should remove clauses dependent on removed clauses in the parent", () => {
```

Comment:
why did we remove those tests, did they become invalid?

Reply:
MLv2 disallows creating queries with invalid clauses, so this case became impossible